include/partition/inertial_flow.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef OSRM_PARTITION_INERTIAL_FLOW_HPP_
+#define OSRM_PARTITION_INERTIAL_FLOW_HPP_
+
+#include "partition/graph_view.hpp"
+#include <vector>
+
+namespace osrm
+{
+namespace partition
+{
+
+class InertialFlow
+{
+  public:
+    InertialFlow(const GraphView &view);
+
+    std::vector<bool> ComputePartition(const double balance, const double source_sink_rate);
```

Comment:
Also seeing you implemented the skeleton as stateful class holding a graph view ref. I suppose you want to abstract the try-n-cuts-take-best inside `ComputePartition` (?)